src/Stratis.Bitcoin.Features.Wallet/WalletManager.cs

```diff
@@ -1524,6 +1571,36 @@ public void RefreshInputKeysLookupLock()
             }
         }
 
+        /// <summary>
+        /// Add to the mapping of transactions kept in memory for faster lookups.
+        /// </summary>
+        private void AddTxLookupLocked(TransactionData transactionData, Transaction transaction)
```

Comment:
If it has locked in the name it should have a comment in remarks section on which lock is required